### ISSUE TITLE
hiding tests?

### ISSUE THREAD

**Issue by seefood** [erwbgy/puppet-check_mk#8]
I'm using this module to monitor Openstack, this means running the agents on both VMs and physical computes. I have lots of false warnings on fs_* that don't exist (but I have no clue what discovers them), IPMI parameters out of range (but I have no clue how to override the threshold settings), Too many Threads (Again, how to ignore or change thresholds?), and many virtual network bridges and interfaces blinking in and out of existence which I want CheckMK to ignore.

In short, I need some better documentation, because grepping through sources and the Check_MK docs is not helping, and I'm a bit lost as to which file creates what config etc.

More docs, or specific clues, please?
